apps/apps.humanatlas.io/src/app/pages/us1/cell-population-predictions/cell-population-predictions.component.ts

```diff
@@ -98,4 +114,22 @@ export class CellPopulationPredictionsComponent {
       this.router.navigate(['/us1']);
     }
   }
+
+  /** Triggered when clicked on download CSV button  */
+  onDownloadCSVButtonClicked() {
+    const headers: string[] = Object.values(this.columnHeaders);
+    const csvData: string[][] = [headers];
+
+    const data = this.dataSource.data;
+    data.forEach((row) => {
```

Comment:
papaparse can serialize arrays of objects so there is no need to turn each object into an array. Simply pass the data array along with config to papaparse, i.e. `papa.unparse(this.predictions(), { columns: this.displayedColumns })`